dcgm/datadog_checks/dcgm/metrics.py

```diff
@@ -0,0 +1,5 @@
+
+METRIC_MAP = {
+    # Exposed OM Metrics to map:
+    'DCGM_FI_DEV_GPU_TEMP': 'dcgm.temperature',
```

Comment:
```suggestion
    'DCGM_FI_DEV_GPU_TEMP': 'temperature',
```

The `namespace` will automatically be added to the metric name 🙂